picard.spec

```diff
@@ -85,8 +100,16 @@ coll = COLLECT(exe,
 
 
 if platform.system() == 'Darwin':
-    info_plist = {'NSHighResolutionCapable': 'True',
-                  'NSPrincipalClass': 'NSApplication'}
+    info_plist = {
+        'NSHighResolutionCapable': 'True',
+        'NSPrincipalClass': 'NSApplication',
+        'CFBundleName': 'Picard',
+        'CFBundleDisplayName': 'MusicBrainz Picard',
+        'CFBundleGetInfoString': 'Audio file tagger',
+        'CFBundleIdentifier': 'org.musicbrainz.picard',
+        'CFBundleVersion': macos_picard_version,
+        'CFBundleShortVersionString': macos_picard_version,
```

Comment:
If I understand the docs correctly the `CFBundleShortVersionString` should only be three parts without beta information. So `CFBundleVersion` currently would be 2.0.4d1 while `CFBundleShortVersionString` would only be `2.0.4`. Other than the docs I found no issue with how the package is handled and the version is displayed in macOS, though.

See:

- https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-111349
- http://hboon.com/difference-between-cfbundleversion-and-cfbundleshortversionstring/

Reply:
Yes, you're right, should be fixed with last commit.